### PR TITLE
docs(dotnet): add docs for SetDefaultExpectTimeout

### PR DESCRIPTION
**Pull request by mxschmitt**
https://github.com/microsoft/playwright-dotnet/issues/2259
https://github.com/microsoft/playwright-dotnet/issues/2603

### CODE REVIEW

File: docs/src/library-csharp.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ You can do the following to leverage Playwright's web-first assertions when you
 using Microsoft.Playwright;
 using static Microsoft.Playwright.Assertions;
 
+// Increasing the default timeout to 10 seconds. Default is 5 seconds.
```
Comment (dgozman) [inline docs/src/library-csharp.md:69]:
```suggestion
// Change the default 5 seconds timeout if you'd like.
```

File: docs/src/test-assertions-csharp-java-python.md [hunk 2]
```diff
@@ -68,6 +68,61 @@ from playwright.sync_api import expect
 expect.set_options(timeout=10_000)
 ```
 
+<Tabs
```
Comment (dgozman) [inline docs/src/test-assertions-csharp-java-python.md:71]: Won't this add empty tabs to the python docs? Perhaps split this section?

File: docs/src/test-assertions-csharp-java-python.md [hunk 3]
```diff
@@ -76,3 +131,7 @@ from playwright.sync_api import expect
 def test_foobar(page: Page) -> None:
     expect(page.get_by_text("Name")).to_be_visible(timeout=10_000)
 ```
+
+```csharp title="UnitTest1.cs"
+Expect(Page.GetByName("Name")).ToBeVisible(new() { Timeout = 10_000 });
```
Comment (dgozman) [inline docs/src/test-assertions-csharp-java-python.md:136]: Copilot detected 😄 

```suggestion
await Expect(Page.GetByText("Name")).ToBeVisibleAsync(new() { Timeout = 10_000 });
```

File: docs/src/intro-csharp.md [hunk 4]
```diff
@@ -91,7 +91,7 @@ Edit the `UnitTest1.cs` file with the code below to create an example end-to-end
 }>
 <TabItem value="nunit">
 
-```csharp
+```csharp title="UnitTest1.cs"
```
Comment (dgozman) [inline docs/src/intro-csharp.md:94]: I wonder why `UnitTest1.cs`, and not `E2ETest.cs` or just `Test1.cs`?

**Review by dgozman — APPROVED** [review]

File: docs/src/intro-csharp.md (hunk 4 above)
Comment (mxschmitt) [reply docs/src/intro-csharp.md:94]: Thats the default `dotnet new` gives you.

**Pull request merged**